flake.nix

```diff
@@ -307,19 +311,22 @@
             version = "0.4.3";
             format = "pyproject";
 
+            # TODO: Upstream patch to use pkg-config for libgssapi-sys
+            # this was needed to allow the crate to find krb5 in newer
+            # versions of macos.
             src = pkgs.fetchFromGitHub {
-              owner = "sfu-db";
+              owner = "LucioFranco";
```

Comment:
I needed to patch libgssapi internally, there should be a way to do this via patching the `Cargo.lock` file but I had trouble since we use sourceRoot. I think its still possible which would allow us to directly depend on the upstream repo rather than my fork.